configuration/helm-charts/ksqldb.md

```diff
@@ -0,0 +1,60 @@
+---
+description: Integrating ksqlDB with Kafbat UI
+---
+
+# Integrating ksqlDB with Kafbat UI
+
+## Summary
+
+This document provides configuration details for integrating ksqlDB with Kafbat UI.
+
+Using Kafbat, you can easily integrate [ksqlDB](https://github.com/confluentinc/ksql) with Kafbat UI. This provides a powerful tool for visualizing and managing real-time streaming data.
+
+![ksqlDB architecture](../../.gitbook/assets/ksqldb-1.png)
+
+[ksqlDB](https://github.com/confluentinc/ksql) is a SQL engine based on Kafka Streams. ksqlDB provides a query layer that allows you to build event streaming applications on Kafka topics. Unlike Kafka Streams, ksqlDB allows you to create new streams or materialized views using SQL.
+
+## Configuration
+
+ksqlDB configuration is specified in `yamlApplicationConfig` section of `values.yaml` file.
+
+```yaml
+# charts/kafka-ui/values.yaml
+yamlApplicationConfig:
+  kafka:
+    clusters:
+      - name: my-kafka-cluster
+        properties:
+          security:
+            protocol: SSL
+        bootstrapServers: b-2.<REDACTED>.<REDACTED>.c3.kafka.ap-northeast-2.amazonaws.com:9094,b-1.<REDACTED>.<REDACTED>.c3.kafka.ap-northeast-2.amazonaws.com:9094
```

Comment:
I believe we can remove (omit) unnecessary properties, unrelated to ksql

Reply:
@Haarolean I removed unrelative values at [9cbdaf9](https://github.com/kafbat/ui-docs/pull/26/commits/9cbdaf904a38e531cb56b0b899a8d2c0acef80fb)